TestPlaybooks/playbook-Fidelis-Test.yml

```diff
@@ -205,11 +214,7 @@ tasks:
       - "7"
     scriptarguments:
       alert_id:
-        complex:
-          root: Fidelis
-          accessor: Alert.ID
-          transformers:
-          - operator: GetFirstObject
+        simple: "2"
```

Comment:
if i recall correctly - alerts are deleted after a while, better to not use const ID